app/static/js/slcManagement.js

```diff
@@ -52,3 +70,58 @@ function withdraw(){
     }
   })
 };
+function slcPrintPDF(courseID){
+  var printProposal = window.open('/serviceLearning/print/' + courseID);
```

Comment:
Could you put a comment in here explaining the firefox vs chrome issues you were having so that someone coming along and trying to 'fix' something or change the timing doesn't repeat your work?